src/styles/components/_override.buttons.scss

```diff
@@ -214,9 +234,21 @@
     }
   }
 }
+/* stylelint-enable */
 
 // Big (landing page) button
 .ds-c-button--big {
   font-family: $montserrat-font;
   font-weight: 600;
-}
\ No newline at end of file
+}
+
+// Help drawer button
+.help-drawer-button {
```

Comment:
There is already a class on the help drawer button `ds-c-help-drawer__toggle` - let's update the class used here and also move it to the helpdrawer override file to be consistent with where this code lives in the core.

Reply:
@babsmiles is there a need to have a version of the help drawer link that works on dark backgrounds?

Reply:
Good catch, I'm sure, it might show up. It might be we could make it so this class, applied to both inverse and transparent buttons could work.

Reply:
@line47 I was able to move the styles and used the `ds-c-help-drawer__toggle` instead. I also added a inverse option. If you add the class `ds-c-button--inverse` to the button it will revert to the inverse option. 

Thank you for the feedback!